addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -280,8 +280,21 @@ def _compute_partner_phone_email(self):
                 applicant.partner_mobile = applicant.partner_id.mobile
 
     def _inverse_partner_email(self):
-        for applicant in self.filtered(lambda a: a.partner_id and a.email_from):
-            applicant.partner_id.email = applicant.email_from
+        for applicant in self:
```

Comment:
This will probably cause issues and here is why

Have you guys checked this? https://github.com/odoo/odoo/commit/4583afc4ec78167efc17035f1ecdc3e7aa3c60b2

and for emails synchronization was brought back because now we blacklist mails..

Reply:
So.. looked again, this will not create problem for future applicants, but it will create an issue for the records that were created before this commig

https://github.com/odoo/odoo/commit/7c0ecbe5a54414c77c475ab93d678f9cad4069c6

Reply:
Hello @sofiagvaladze 

We are not creating a partner for an applicant anywhere until we hire them or schedule a meeting with them
But the partner is being created upon an email sent (Application acknowledge) with only name and email (Mail system will create one with name and email only)

I've another task in which we do not want to send this email anymore( [Task](https://www.odoo.com/web/#id=3636002&cids=2&menu_id=6478&action=4043&model=project.task&view_type=form) )
Link:https://i.imgur.com/LVlIOpe.png

Thus partner is not created anymore until we hire them  or schedule a meeting with them
If we need to sync partner with the applicant
Should we create a partner for each applicant with the blacklist mail flow?

Thank you for looking into it :innocent: